test/models/PostponedVatStatementGroupSpec.scala

```diff
@@ -28,48 +28,55 @@ class PostponedVatStatementGroupSpec extends SpecBase {
-      val currentDate = LocalDate.of(2023,10,16)
-      when(mockDateTimeService.systemDateTime())
-        .thenReturn(currentDate.atStartOfDay())
-      PostponedVatStatementGroup(dateOfPreviousMonth, Seq())(messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
+      val currentDate = LocalDate.of(2023, 10, 16)
+
+      when(mockDateTimeService.systemDateTime()).thenReturn(currentDate.atStartOfDay())
+      PostponedVatStatementGroup(dateOfPreviousMonth, Seq())(
+        messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
     }
 
     "return false if statement date is of previous month and is accessed on or before " +
       "14th day of the current month" in new Setup {
       when(mockDateTimeService.systemDateTime())
         .thenReturn(date.atStartOfDay())
       val dateOfPreviousMonth: LocalDate = date.minusMonths(1).withDayOfMonth(10)
-      PostponedVatStatementGroup(dateOfPreviousMonth, Seq())(messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe false
+
+      PostponedVatStatementGroup(dateOfPreviousMonth, Seq())(
+        messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe false
     }
 
     "return true if statement date is not from the previous month" in new Setup {
       when(mockDateTimeService.systemDateTime())
         .thenReturn(date.atStartOfDay())
       val dateOfCurrentMonth: LocalDate = date.withDayOfMonth(16)
-      PostponedVatStatementGroup(dateOfCurrentMonth, Seq())(messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
+
+      PostponedVatStatementGroup(dateOfCurrentMonth, Seq())(
+        messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
     }
 
-    //TODO: To be fixed, has been ignored temporarily as causing build failure
     "return true if statement date is not from the previous month and accessed " +
-      "before 15th day of current date " ignore new Setup {
-      when(mockDateTimeService.systemDateTime())
-        .thenReturn(date.atStartOfDay())
+      "before 15th day of current date " in new Setup {
+
+      private val isCurrentDayBefore15 = LocalDate.now.getDayOfMonth < 15
+
+      if (isCurrentDayBefore15) when(mockDateTimeService.systemDateTime()).thenReturn(date.atStartOfDay())
+
       val dateOfCurrentMonthAndBefore15ThDay: LocalDate = date.withDayOfMonth(12)
-      if (LocalDate.now.getDayOfMonth < 15) {
+
+      if (isCurrentDayBefore15) {
         PostponedVatStatementGroup(
           dateOfCurrentMonthAndBefore15ThDay,
-          Seq())(messages(app),mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
+          Seq())(messages(app), mockDateTimeService).isPreviousMonthAndAfter14Th mustBe true
       }
     }
   }
 
   trait Setup {
     val mockDateTimeService: DateTimeService = mock[DateTimeService]
-    val date: LocalDate = LocalDate.of(2023,10,1)
+    val date: LocalDate = LocalDate.of(2023, 10, 1)
```

Comment:
curious why add spacing to the dates ?

Reply:
It (single space after ,)  should be there as per correct/standard formatting for the arguments passed in the methods /functions /classes . Earlier it was incorrect, hence it has been rectified.